src/Sylius/Behat/Context/Setup/CatalogPromotionContext.php

```diff
@@ -682,6 +792,9 @@ private function createCatalogPromotion(
         $catalogPromotion = $this->catalogPromotionExampleFactory->create([
             'name' => $name,
             'code' => $code,
+            'startDate' => $startDate,
```

Comment:
I made sure that we use cammelCase syntax.
For instance, in:
`src/Sylius/Behat/Context/Setup/PaymentContext.php`

Reply:
So, we are using two different conventions: https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Fixture/ChannelFixture.php 😢